_includes/trackers_index_response.json

```diff
@@ -0,0 +1,65 @@
+{
+  "trackers" : [
```

Comment:
Should we sort this alphabetically as well?

And if `to` and `from` addresses are given, they will be returned as well.